<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BuddySystem-1.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout1"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Variables:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>true</bool>
        </property>
        <attribute name="headerCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>77</number>
        </attribute>
        <column>
         <property name="text">
          <string>Nombre</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Reservado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ocupado</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Historial:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>LÃ­nea de comandos:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menuA_yuda">
    <property name="title">
     <string>A&amp;yuda</string>
    </property>
    <addaction name="actionHelpHelp"/>
    <addaction name="separator"/>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menu_Archivo"/>
   <addaction name="menuA_yuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileExit">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/file-exit.png</normaloff>:/file-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/help-about.png</normaloff>:/help-about.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Acerca de ...</string>
   </property>
  </action>
  <action name="actionHelpHelp">
   <property name="icon">
    <iconset resource="../res/resources.qrc">
     <normaloff>:/help-help.png</normaloff>:/help-help.png</iconset>
   </property>
   <property name="text">
    <string>A&amp;yuda</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFileExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotFileExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelpHelp</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotHelpHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHelpAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotHelpAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotFileExit()</slot>
  <slot>slotHelpHelp()</slot>
  <slot>slotHelpAbout()</slot>
 </slots>
</ui>
